### PR TITLE
Add index.html and styles.css

### PR DESCRIPTION
**Pull request by WilfriedM01**
-  Copied code from the trial project  for it to be reviewed (html and css files)
-  Added form features

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+  </head>
```
Comment (WilfriedM01) [inline index.html:7]: Please add the `<meta/>` tag `viewport` in the header of your page, it allows the browser to control the dimension and scaling.

Comment (WilfriedM01) [inline index.html:7]: **- [ ] Required:** Please add the `<meta/>` tag `viewport` in the `<head></head>` of your page, it allows the browser to control the dimensions and scaling of the page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        </div>
+        <div class="form-example">
+            <textarea name="Comment" cols="30" rows="10" placeholder="Write here your comment"></textarea>
+        </div>
+        <div class="form-example">
+            <button id="submit">Submit</button>
+
+        </div>
+
+    </form>
```
Comment (WilfriedM01) [inline index.html:56]: - [ ] **Required:** 

1. There are some empty lines in your code, please remove them and your code be clean;
2. Please use the semantic tags on your page to increase the SEO of your page. Please, wrap your form into the section tag `<section></section/>`. It is better to wrap a generic section of the page into a `<section>` tag for best practices.

**Review by WilfriedM01 — COMMENTED** [review]
Hi @WilfriedM01 

**Great job but there are minor issues that need your attention, please.**

**Highlights**

- You well document your repo :+1: 

- You add a professional README file :+1: 

**Required Changes :recycle:** 
1. You do not use linters on your project. take time to setup them
2. Check the below comments 

Thanks :v:

File: index.html (hunk 2 above)
Comment (Shedrack-Sunday) [reply index.html:56]: - [x] Brilliant review you gave her 👏 . However, just as you gave a reason in your earlier review, kindly give a "why" on why a `section` tag is necessary here. You might have it from your perspective that it's for semantic reasons, but imagine you're reviewing another developer, and stating a reason for them to take such action. Welldone 👏

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @WilfriedM01         👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of GitHub features ✔️   
- Nice and respectful review given 👍 

  
# Required Changes  🌵 

- [x] You made some impressive reviews in the comments, which I must say is a job well done. However, there's one you missed 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Strangeal — APPROVED** [review]
Hi @WilfriedM01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**